patterns/multi-node-vllm/helm.tf

```diff
@@ -0,0 +1,78 @@
+################################################################################
+# Device Plugin(s)
+################################################################################
+
+resource "helm_release" "nvidia_device_plugin" {
+  name             = "nvidia-device-plugin"
+  repository       = "https://nvidia.github.io/k8s-device-plugin"
+  chart            = "nvidia-device-plugin"
+  version          = "0.17.0"
+  namespace        = "nvidia-device-plugin"
+  create_namespace = true
+  wait             = false
+}
+
+resource "helm_release" "aws_efa_device_plugin" {
+  name       = "aws-efa-k8s-device-plugin"
+  repository = "https://aws.github.io/eks-charts"
+  chart      = "aws-efa-k8s-device-plugin"
+  version    = "v0.5.7"
+  namespace  = "kube-system"
+  wait       = false
+
+  values = [
+    <<-EOT
+      nodeSelector:
+        vpc.amazonaws.com/efa.present: 'true'
+      tolerations:
+        - key: nvidia.com/gpu
+          operator: Exists
+          effect: NoSchedule
+    EOT
+  ]
+}
+
+################################################################################
+# Kuberay (Ray.io)
+################################################################################
+
+resource "helm_release" "kuberay_operator" {
+  namespace        = "kuberay-operator"
+  create_namespace = true
+  name             = "kuberay-operator"
+  repository       = "https://ray-project.github.io/kuberay-helm/"
+  chart            = "kuberay-operator"
+  version          = "1.2.2"
+}
+
+resource "helm_release" "kuberay_cluster" {
+  namespace        = "ray-cluster"
+  create_namespace = true
+  name             = "ray-cluster"
+  repository       = "https://ray-project.github.io/kuberay-helm/"
+  chart            = "ray-cluster"
+  version          = "1.2.2"
+}
```

Comment:
I don't think you need these. From what I can tell its managing the ray cluster internally.

Reply:
let me try without - just to be clear, you're saying without both kuberay and ray-cluster, or just without ray-cluster?

Reply:
Yeah remove both. `kuberay-operator` CRUDs the CRDs invoked by `ray-cluster`, which is not happening with lws. Its doing a naive implementation of ray cluster via the `ray init` commands. Which also means it is not fault tolerant by default because Ray clusters should be using an external redis store for its state. We will need to dig a bit deeper on what happens if the LWS manager crashes because of an OOM or something.

Reply:
you were right! was able to remove both the operator and the cluster charts and everything still works as expected. updated in https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/2072/commits/f4229bc7ee746578ced5edf39084fd47e9668067